src/adapters/adapter.js

```diff
@@ -24,6 +42,17 @@ class Adapter {
 
     opts.encodedBranch = opts.encodedBranch || encodeURIComponent(decodeURIComponent(repo.branch));
 
+    const host =
+      location.protocol + '//' + (location.host === 'github.com' ? 'api.github.com' : location.host + '/api/v3');
+    const CACHE_KEY = `${host}/repos/${opts.repo.username}/${opts.repo.reponame}${path || ''}`;
+    const CACHE_NAME = 'loaded-tree';
+    const STORE_NAME = 'loaded-tree-storage';
+    let force;
+    if(location.hash.search('frc=') !== -1){
```

Comment:
This is nice but we will need something in the UI for users to trigger this. Maybe add an icon in place of the loading icon to do trigger refresh.

Reply:
Agree, UI is much better.